airbyte-integrations/bases/debezium/src/main/java/io/airbyte/integrations/debezium/internals/MSSQLConverter.java

```diff
@@ -26,12 +33,17 @@ public void configure(Properties props) {}
   @Override
   public void converterFor(final RelationalColumn field,
                            final ConverterRegistration<SchemaBuilder> registration) {
-    if (SMALLDATETIME_TYPE.equalsIgnoreCase(field.typeName())) {
+    if (DATE_TYPE.equalsIgnoreCase(field.typeName())
```

Comment:
can we use Set of date-related fields instead?

Reply:
There is no way to get a native Mssql enum but updated it to use some array. It should look better now.
Thanks!